utils/convert-gamess-ecp-to-qmcpack

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import sys,os
+sys.path.append(os.path.join(os.path.dirname(__file__), '../nexus/lib'))
```

Comment:
In case someone moves the exe and tries to use it, perhaps add this guard:

```
nexus_lib = os.path.join(os.path.dirname(__file__), '../nexus/lib')
sys.path.append(nexus_lib)
try:
    from pseudopotential import GaussianPP
except:
    print('Nexus must be installed to run this script.\nExpected Nexus library at {}'.format(nexus_lib))
    sys.exit(1)
#end try
```
The script will run without error when either 1) the script has not been moved, 2) Nexus has been installed.  This at least tells them what to do if they have made this mistake instead of just having an `ImportError` printed.

Reply:
Thank you, Jaron. This looks good.

Reply:
Commit pushed. All 3 cases work as expected:
1. unmoved exe works successfully
2. moved exe with Nexus installed works successfully
3. mved exe without Nexus installed fails with verbose message "Nexus must be installed ..."